<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RepoWindow</class>
 <widget class="QMainWindow" name="RepoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RepoWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="RemoteConfig" name="remoteConfig" native="true"/>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_direct">
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator:unchecked {
     image: url(:/Icon_Slider_Off.png);
 }
QCheckBox::indicator:checked {
     image: url(:/Icon_Slider_On.png);
 }</string>
      </property>
      <property name="text">
       <string>Direct Mode</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
      <item>
       <widget class="QCheckBox" name="checkBox_assistant">
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator:unchecked {
     image: url(:/Icon_Slider_Off.png);
 }
QCheckBox::indicator:checked {
     image: url(:/Icon_Slider_On.png);
 }</string>
        </property>
        <property name="text">
         <string>Assistant </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_webapp">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Run WebApp</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_autostart">
      <property name="text">
       <string>start assistant at log-in</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Global Settings :</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Number of copies :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBox_numcopies"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>476</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRefresh">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RemoteConfig</class>
   <extends>QWidget</extends>
   <header>remoteconfig.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_assistant</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_webapp</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_assistant</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBox_direct</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
